src/styles/blocks/header.scss

```diff
@@ -0,0 +1,99 @@
+* {
+  padding: 0;
+  margin: 0;
+  list-style: none;
+  box-sizing: border-box;
+}
+
+a {
+  text-decoration: none;
+  color: inherit;
+}
+
+html {
+  font-size: 16px;
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+}
+
+:root {
+  --header-height: 60px;
+}
+
+.roboto-medium {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-style: normal;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  margin: 0 auto;
+  padding: 0 50px;
+  justify-content: space-between;
+  width: 100%;
+}
+
+.logo {
+  display: block;
+  height: 40px;
+  width: 40px;
+}
+
+.nav {
+  text-transform: uppercase;
+}
```

Comment:
The text transformation for `.nav` is applied to all nested elements. If this is the only style for `.nav`, consider whether a class is necessary, or if the styles could be applied directly to `.nav__list` or `.nav__item`.